specs/windows/windows_search.table

```diff
@@ -0,0 +1,18 @@
+table_name("windows_search")
+description("Run searches against the windows system index database.")
+schema([
+    Column("path", TEXT, "path of item"),
+    Column("attribute", TEXT, "attribute"),
+    Column("value", TEXT, "value"),
+    Column("max_results", INTEGER, "maximum number of results returned by windows api", additional=True),
+    Column("sort", TEXT, "sort for windows api", additional=True),
+    Column("select_columns", TEXT, "columns to include", additional=True),
+    Column("query", TEXT, "windows search query", additional=True),
```

Comment:
I would probably hide these. Hidden columns do not appear in the output of a `select *`

Reply:
done